### DISCUSSION TITLE
Windows insider builds

### DISCUSSION THREAD

**Discussion by jcasale** [VSCodium/vscodium#1367] [General]
I was hoping to gain insight into the process behind generating Windows insider builds. They appear to be created manually and I don't know what they depend on.

I have started watching them in the hopes of moving off of the upstream release's daily updates. This version of insiders has significant benefits for my use case due to the packaging, but they don't appear to be produced consistently.

Does anyone know the details behind them?

Thanks!

**Comment by daiyam** [comment]
They are produced by workflows (for example: https://github.com/VSCodium/vscodium/blob/master/.github/workflows/insider-windows.yml).
But [insider-spearhead.yml](https://github.com/VSCodium/vscodium/blob/master/.github/workflows/insider-spearhead.yml) run first to check if the build with the newest version is successful (since they are adding changes that can break the different [patches](https://github.com/VSCodium/vscodium/tree/master/patches)) and update [insider.json](https://github.com/VSCodium/vscodium/blob/master/insider.json) which is then used by `insider-linux.yml`, `insider-macos.yml` and `insider-windows.yml`

**Comment by jcasale** [comment]
Oh so the builds are scheduled daily, thank you for the detailed explanation.
